x-pack/plugins/uptime/server/lib/requests/get_journey_steps.ts

```diff
@@ -53,28 +71,45 @@ export const getJourneySteps: UMElasticsearchQueryFn<GetJourneyStepsParams, Ping
       { '@timestamp': { order: 'asc' } },
     ] as const),
     _source: {
-      excludes: ['synthetics.blob'],
+      excludes: ['synthetics.blob', 'screenshot_ref'],
     },
     size: 500,
   };
   const { body: result } = await uptimeEsClient.search({ body: params });
 
-  const screenshotIndexes: number[] = (result.hits.hits as Array<SearchHit<Ping>>)
-    .filter((h) => h._source?.synthetics?.type === 'step/screenshot')
-    .map((h) => h._source?.synthetics?.step?.index as number);
+  const steps = parseResult(
+    result.hits.hits.map(({ _id, _source }) => Object.assign({ _id }, _source)),
```

Comment:
The `JourneyStepType` expects `_id` to be nested with the fields from `_source`. This is something we could adjust if we want, to avoid adding `map` latency to requests that parse this type of data.